File: network_data.js
const fs = require('fs');
const axios = require('axios');

// Read JSON file
const data = JSON.parse(fs.readFileSync('network_data.json', 'utf-8'));

// Send POST request
axios.post('http://localhost:3000/api/usage', data)
  .then(response => {
    console.log(response.status, response.data);
  })
  .catch(error => {
    console.error('Error:', error.response?.data || error.message);
  });

File: network_data.py
import json
import requests

# Read from the JSON file
with open("network_usage.json", "r") as file:
    data = json.load(file)

# Send the data to your Next.js API
response = requests.post("http://localhost:3000/api/usage", json=data)

# Print the result
print(response.status_code)
print(response.json())
